polygon/bridge/bridge_test.go

```diff
@@ -137,13 +137,6 @@ func TestBridge(t *testing.T) {
 	require.Equal(t, event1Data, rlp.RawValue(res[0].Data())) // check data fields
 	require.Equal(t, event2Data, rlp.RawValue(res[1].Data()))
 
-	res, err = b.Events(ctx, 4)
```

Comment:
what happened here? these asserts are no longer needed or it's unintentional?

Reply:
Was a duplicate.

https://github.com/erigontech/erigon/blob/c14ad5fe396b22755fad5f93a4a2b074ae6bf888/polygon/bridge/bridge_test.go#L127-L138